libs/deal-ticket/src/components/trading-mode-tooltip/trading-mode-tooltip.tsx

```diff
@@ -0,0 +1,93 @@
+import { t } from '@vegaprotocol/react-helpers';
+import { MarketTradingMode, AuctionTrigger } from '@vegaprotocol/types';
+import { ExternalLink } from '@vegaprotocol/ui-toolkit';
+import type { ReactNode } from 'react';
+import { MarketDataGrid } from './market-data-grid';
+
+type TradingModeTooltipProps = {
+  market: {
+    tradingMode: MarketTradingMode;
+    data: { trigger: AuctionTrigger | null } | null;
+  };
+  compiledGrid: { label: ReactNode; value?: ReactNode }[];
+};
+
+export const TradingModeTooltip = ({
```

Comment:
moved trading-mode-tooltip so that we can re-use it. (in trading app and deal-ticket lib)